lib/zstd/src/time.c

```diff
@@ -0,0 +1,9 @@
+#include <sys/time.h>
+#include <zephyr.h>
+
+int _gettimeofday(struct timeval *tv, void *tzvp) {
```

Comment:
Does something need doing with `tzvp` even if it's not being used on this system i.e. should be be being set to zero values.

Additionally, should this be an implementation of `gettimeofday` rather than `_gettimeofday`?